utilities_common/db.py

```diff
@@ -11,7 +11,17 @@ def __init__(self):
         self.cfgdb = ConfigDBConnector()
         self.cfgdb.connect()
         self.db = SonicV2Connector(host="127.0.0.1")
-        for db_id in self.db.get_db_list():
+
+        # Skip connecting to chassis databases in line cards
+        db_list = list(self.db.get_db_list())
+        if not device_info.is_supervisor():
+            try:
+                db_list.remove('CHASSIS_APP_DB')
```

Comment:
Similar comment, we can have a function in py_common/multi_asic.py or py_common/deviceinfo to get the list of chassis db instances and use here. Will help in future when we add more DBs